### PR TITLE
VIITE-2331 update and fix swagger configuration

### PR DESCRIPTION
**Pull request by peltsi212**

### CODE REVIEW

File: digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala [hunk 1]
```diff
@@ -38,29 +57,66 @@ class SearchApi(roadAddressService: RoadAddressService) extends  ScalatraServlet
...
     time(logger, "GET request for /road_numbers?") {
       roadAddressService.getRoadNumbers
     }
   }
 
-  get("/road_address/:road/?") {
+  private val getRoadAddressWithRoadNumber: SwaggerSupportSyntax.OperationBuilder = (
+    apiOperation[List[Map[String, Any]]]("getRoadAddressWithRoadNumber")
+      .parameters(
+        pathParam[Long]("road").description("roadNumber")
```
Comment (skosonen) [inline digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala:75]: Description could be more human friendly.

File: digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala [hunk 2]
```diff
@@ -38,29 +57,66 @@ class SearchApi(roadAddressService: RoadAddressService) extends  ScalatraServlet
...
       val trackCodes = multiParams.getOrElse("tracks", Seq()).map(_.toInt)
       roadAddressService.getRoadAddressWithRoadNumber(roadNumber, Track.applyAll(trackCodes)).map(roadAddressMapper)
     }
   }
 
-  get("/road_address/:road/:roadPart/?") {
+  private val getRoadAddressesFiltered: SwaggerSupportSyntax.OperationBuilder = (
+    apiOperation[List[Map[String, Any]]]("getRoadAddressesFiltered")
+      .parameters(
+        pathParam[Long]("road").description("roadNumber"),
```
Comment (skosonen) [inline digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala:92]: These descriptions could also be more human friendly

File: digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala (hunk 1 above)
Comment (peltsi212) [reply digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala:75]: I do not know application very well. If I make more human friendly description I create it by guessing. I tested each api and they work and live result is documentation itself. This is what i could do with original 1 storypoint limitation.

Comment (peltsi212) [reply digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala]: fixed

File: digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala (hunk 2 above)
Comment (peltsi212) [reply digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/SearchApi.scala]: fixed

**Pull request merged**